lte/gateway/python/magma/mobilityd/mac.py

```diff
@@ -36,7 +36,7 @@ def as_redis_key(self, vlan: str) -> str:
         """
         key = str(self.mac_address).replace(':', '_').lower()
         if vlan:
-            return vlan + "." + key
+            return "v{}.{}".format(vlan, key)
```

Comment:
nit: we're trying to follow lazy load formatting pattern for strings, can you update this to:

` return str("v%s.%s", (vlan, key)) ` just to be consistent

Reply:
sounds good. since this is backport, I do not want to change this patch. I will send separate patch for master branch.